_authors/noashoval.markdown

```diff
@@ -0,0 +1,7 @@
+---
+name: Noa Shoval
+short_name:Noa
```

Comment:
nit: Please add a space between `short_name:` and your name.

Reply:
added the space!